src/core_modules/capture-core/hooks/useTrackedEntityTypesWithCorrelatedPrograms.js

```diff
@@ -1,7 +1,8 @@
 // @flow
```

Comment:
We're currently pushing to remove these programCollection objects as part of the plugin-epic. The registration page is one of the places we have rewritten in https://github.com/dhis2/capture-app/pull/3200 and we should probably align these PR's.